src/main/java/io/xpring/xrpl/model/XRPMemo.java

```diff
@@ -19,34 +17,36 @@
   }
 
   /**
-   * An arbitrary hex value, conventionally containing the content of the memo.
+   * (Optional) An arbitrary hex value, conventionally containing the content of the memo.
```

Comment:
Is this 'optional'? It just returns a 0 byte if there's no data?

Reply:
So this is optional in the sense that it doesn't have to be present, but it's not an `Optional` type because according to Neil you don't typically use `Optional`s in conjunction with `Collection`s, you just return an empty `Collection`... Noah suggested the default byte array values - and `new byte[0]` returns an empty byte array of length 0, not the 0 byte.

Reply:
Ah, I read this as an array with a single value, 0x00. Sure, this makes sense.